aspdotnet/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,28 @@
+name: ASP.NET
+files:
+- name: aspdotnet.yaml
+  options:
+  - template: init_config
+    options:
+      - template: init_config/default
+  - template: instances
+    options:
+    - name: host
+      description: |
+        ASP .NET host to connect to.
+        "." means the current host, any other value will have the Agent attempt to connect to a remote host.
+        Note: Remote access requires additional permissions -
+        https://support.microsoft.com/en-us/help/922775/how-to-troubleshoot-monitoring-and-logging-issues-for-performance-coun  /noqa
```

Comment:
🤔 This URL 404's.  The shorter version https://support.microsoft.com/en-us/help/922775 does too.

Reply:
Its not the first time this happens with ms urls -.-